apps/web/src/components/Profile/components/WorkPreferences/FormFields.tsx

```diff
@@ -77,7 +77,16 @@ const FormFields = ({ labels }: FormFieldProps) => {
           legend={labels.locationPreferences}
           name="locationPreferences"
           id="locationPreferences"
-          items={enumToOptions(WorkRegion).map(({ value }) => ({
+          items={enumToOptions(WorkRegion, [
+            WorkRegion.Telework,
+            WorkRegion.NationalCapital,
+            WorkRegion.Atlantic,
+            WorkRegion.Quebec,
+            WorkRegion.Ontario,
+            WorkRegion.North,
+            WorkRegion.Prairie,
+            WorkRegion.BritishColumbia,
+          ]).map(({ value }) => ({
```

Comment:
I'm wondering if it is worth moving this to a utility function since we are using it in 4 different places.

Reply:
Yeah, makes sense definitely

Factored out in https://github.com/GCTC-NTGC/gc-digital-talent/pull/7299/commits/a7aff60b1b5510900eff12db214022ff9c300e4a
Translations added in https://github.com/GCTC-NTGC/gc-digital-talent/pull/7299/commits/b093a496dd2ffa8f11f5cc121c2b74864e627f9d